SwrveCommonSDK/src/main/java/com/swrve/sdk/SwrveImp.java

```diff
@@ -1182,7 +1189,7 @@ protected void startCampaignsAndResourcesTimer(boolean sessionStart) {
             public void run() {
                 checkForCampaignAndResourcesUpdates();
             }
-        }, campaignsAndResourcesFlushRefreshDelay.longValue(), campaignsAndResourcesFlushFrequency.longValue(), TimeUnit.MILLISECONDS);
+        }, 0l, campaignsAndResourcesFlushFrequency.longValue(), TimeUnit.MILLISECONDS);
```

Comment:
Because we want to start the campaignsAndResourcesExecutor straight away. There's no need for it to add a delay here. Let the delay happen directly after queued events (and refresh resources) have been sent inside of checkForCampaignAndResourcesUpdates method. 

Essentially this whole pull request is about movign the delay from starting the campaignsAndResourcesExecutor to inside the checkForCampaignAndResourcesUpdates.